spec/spec_helper.rb

```diff
@@ -1,15 +1,26 @@
 # frozen_string_literal: true
 
+require 'simplecov'
+require 'codecov'
+SimpleCov.start do
+  add_filter '/spec/'
+end
+
+SimpleCov.minimum_coverage 95
+SimpleCov.formatter = SimpleCov::Formatter::Codecov if ENV['TRAVIS']
```

Comment:
```suggestion
SimpleCov.formatter = SimpleCov::Formatter::Codecov if ENV['CI']
```
just sligghtlyy more generic lol (I did the same for CI detection in the other SDKs)